OpenRA.Mods.Common/Traits/Upgrades/DeployToUpgrade.cs

```diff
@@ -32,6 +34,18 @@ public class DeployToUpgradeInfo : ITraitInfo, Requires<UpgradeManagerInfo>
 		[Desc("Cursor to display when unable to (un)deploy the actor.")]
 		public readonly string DeployBlockedCursor = "deploy-blocked";
 
+		[Desc("Facing that the actor must face before deploying. Will deploy disregarding facing by default.")]
+		public readonly int? Facing = null;
```

Comment:
Yeah that is a problem which I noticed when testing the suggestion. Will revert to -1  :/